File: leetcode/easy/Palindrome Number.js
var isPalindrome = function(x) {
    let string = String(x)
    let array = string.split("")
    for(let i=0;i<array.length;i++) {
        let reverse = array.length-1
                if(array[i]!==array[reverse-i]) {
                    return false
                }else if(i===array.length-1) {
                    return true
                }
    }
};

File: leetcode/easy/python/Palindrome Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0: return False
        print(str(x)[::-1])
        return str(x) == str(x)[::-1] #[::]で範囲全指定,[::-1]にすることで反転される
                                      #元の引数と反転した物が一致していればtrue